store/data/projects.yaml

```diff
@@ -1036,3 +1036,28 @@
   introLogoVisible: true
   gitHubRepoName: treeverse/lakeFS
   youTubeVideoId: xThorxDzmrw
```

Comment:
shouldn't this be emptied or changed to the official one?

Reply:
vide url for OVMS is in the last line: youTubeVideoId: AfytPrAVdfc

Reply:
So shouldn't we remove this line?

Reply:
No this is part of previous project not ovms.